packages/form/src/Default/stories/Default-index.stories.tsx

```diff
@@ -33,6 +33,7 @@ export const Default = (args: FormDefaultProps) => {
     {
```

Comment:
remove the `key` prop from these items.

change it with an `id` instead (and make it optional in the type).

In the `renderFormElements` fn check for the existence of the `id` prop, if it doesn't exist, use the index of the element that you get from the `map`.